### DISCUSSION TITLE
Add navigation buttons within the Album view

### DISCUSSION THREAD

**Discussion by Minstrel27** [navidrome/navidrome#2411] [Ideas]
When opening a specific album within the artist view, it would be nice if we could navigate to previous or next from that view.  Currently we have to go back to the Artist view, then click on the next album to see the details.

![afT3HcmwiI](https://github.com/navidrome/navidrome/assets/84980328/b1f1359d-0bdf-4a9b-b3e2-3f70b71abd4a)

**Comment by metalheim** [comment]
Just some thoughts: What is "next" album?
Next album by year? By release date? What if these are different/conflict?

**Reply by dweymouth** [reply]
And it would depend on which screen you came from I guess? ie if clicking on an album from the Albums page with random sorting, the Next/Prev buttons should paginate that same random sort? Yeah I don't see how this feature could be consistently implemented

**Reply by Minstrel27** [reply]
The "next" album would be however it's sorted by.  I believe the default is by the date tag (at least that's how it appears to me) so clicking on next or previous would display whatever record was listed before or after the one currently on.

For example, if I'm looking at the releases by a specific artist (IE, RUSH)



![firefox_1Rjf8cv5au](https://github.com/navidrome/navidrome/assets/84980328/7b27c36f-2c2f-4d38-8f13-a72a6a4877e9)

I click on one of the releases (example, 2112)


![firefox_J5HouWdHdQ](https://github.com/navidrome/navidrome/assets/84980328/69853926-718c-4c37-9331-d488a4ec6f1f)

That displays the view of that record with all the tracks.   Clicking Next here would bring me to "All The Worlds a Stage" since that was the next in the sorted list, and Previous would bring me to "Caress of Steel" since that was the previous record in the sorted list.

I don't think this feature has much value in the random sort, but would be very helpful when looking at a specific artist.
